inspirehep/config.py

```diff
@@ -443,6 +455,10 @@
 
 SEARCH_ELASTIC_DEFAULT_INDEX = 'hep'
 
+INSPIRE_PATH = pkg_resources.resource_filename("inspirehep", "")
```

Comment:
Uh. I thought you wanted to use `INSPIRE_PATH` afterwards into `JSON_SCHEMA_PATHS`. In which other place do you plan to use it?